data/src/main/java/com/stop/data/remote/model/nowlocation/BusInfo.kt

```diff
@@ -0,0 +1,30 @@
+package com.stop.data.remote.model.nowlocation
+
+import com.squareup.moshi.Json
+import com.stop.data.model.nowlocation.BusInfoRepositoryItem
+
+data class BusInfo(
+    @Json(name = "stopFlag")
+    val isArrivedAtStation: String,
+    val sectionId: String,
+    @Json(name = "gpsX")
+    val longitude: String,
+    @Json(name = "gpsY")
+    val latitude: String,
+    @Json(name = "vehId")
+    val busId: String,
+    @Json(name = "plainNo")
+    val busNumber: String,
+    @Json(name = "isrunyn")
+    val isRun: String,
+) {
+    fun toRepositoryModel() = BusInfoRepositoryItem(
+        isArrivedAtStation,
+        sectionId,
+        latitude,
+        longitude,
+        busId,
+        busNumber,
+        isRun
+    )
```

Comment:
```suggestion
    fun toRepositoryModel() = BusInfoRepositoryItem(
        isArrivedAtStation = isArrivedAtStation,
        sectionId = sectionId,
        latitude = latitude,
        longitude = longitude,
        busId = busId,
        busNumber = busNumber,
        isRun = isRun,
    )
```

데이터 클래스의 파라미터가 많을 때, 파라미터 이름을 적어주면 가독성이 좋아지고
파라미터가 추가되었을 때, 작업하기도 편리합니다.